packages/pluggableWidgets/activity-indicator-native/src/ActivityIndicator.xml

```diff
@@ -2,7 +2,8 @@
+    <studioProCategory>Display</studioProCategory>
```

Comment:
I think the new image file should override the static image set here. Leaving it in would help with backwards compatibility. Again, please check.